Jenkinsfile

```diff
@@ -578,20 +578,20 @@ pipeline {
                         buildHipClangJobAndReboot( setup_flags: Tensile_setup + extra_log_env + Bf16_flags, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx908:xnack-", miotensile_version: Tensile_version, target_id: "ON")
                     }
                 }
-                // stage('Fp32 Hip Tensile-Latest All gfx90a') {
-                //     when { expression { params.SMOKE_MIOPENTENSILE_LATEST && !params.DISABLE_ALL_STAGES } }
-                //     agent{ label rocmnode("gfx90a") }
-                //     steps{
-                //         buildHipClangJobAndReboot( setup_flags: Tensile_setup + extra_log_env, build_env: Tensile_build_env + extra_log_env, gpu_arch: "gfx90a:xnack-", test_flags: ' --verbose ', miotensile_version: Tensile_version, target_id: "ON")
-                //     }
-                // }
-                // stage('Bf16 Hip Tensile-Latest All gfx90a') {
-                //     when { expression { params.SMOKE_MIOPENTENSILE_LATEST && !params.DISABLE_ALL_STAGES } }
-                //     agent{ label rocmnode("gfx90a") }
-                //     steps{
-                //         buildHipClangJobAndReboot( setup_flags: Tensile_setup + extra_log_env + Bf16_flags, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx90a:xnack-", miotensile_version: Tensile_version, target_id: "ON")
-                //     }
-                // }
+                stage('Fp32 Hip Tensile-Latest All gfx90a') {
```

Comment:
Remove `All` from the stage name (or replace it with `Smoke`). `All` means full test (`-DMIOPEN_TEST_ALL=On`), but this stage is intended to perform smoke testing.

Ditto for **_all stage names_** that reside within `Smoke MIOpenTensile Latest`.

Please refer to this: https://github.com/ROCmSoftwarePlatform/MIOpen/blob/1df9a07991727f1ac76d9507e963f4fe047eb4b8/Jenkinsfile#L208-L211

Reply:
Is it so that _non-latest_ MIOpenTensile does not support gfx90a? The block of code starting from line 736 (`Fp32 Hip Tensile All gfx90a`) is still commented out.

Reply:
1. will do.
2. non-latest miopentensile use tips of official released version, so yes, it's not supporting gfx90a at this point.